<launch>
    <node pkg="drive" type="mgeo_pub.py" name="mgeo_pub"  />

    <node pkg="drive" type="global_path_dijkstra.py" name="global_path"  />
    <node pkg="drive" type="gpsimu_parser.py" name="GPS_IMU_parser" />

    <node pkg="drive" type="local_path_pub.py" name="local_path"  />

    <node pkg="drive" type="lattice_planner.py" name="lattice_planner" output="screen"/>
    <node pkg="drive" type="pure_pursuit_m2.py" name="pure_pursuit" output="screen"/>

    <node pkg="drive" type="tf_pub.py" name="tf"  />
    <node pkg="rviz" type="rviz" name="rviz" args="-d $(find drive)/rviz/m2.rviz"/>
</launch>
